File: functions/notes.py
# Robert Nelson, Functions Notes Python
"""*/

def add():
    numOne = int(input("Please give me a number:\n"))
    numTwo = int(input("Please give me another number:\n"))

    print(numOne+numTwo)

add()
add()
add()



/*""""""
def add():

    number = int(input("Please give me a number:\n"))

#def add(numOne, numTwo):#parameters set the name of the variable (just fot eh function)
 #   print(numOne+numTwo)

add(number, 21)#arguements set the value of the variable just for that instance of the function
addition = add(6,4)
add(addition, 1)
add()
add()"""
"""
def values(type):
    return input(f"please give me a {type}:\n")
name = values("name")
print(f"{name} was really fast")
"""
def temp():
    celcius = int(input("What tempretaure is it outside in Farenheit for you right now?"))
    print(celcius)
cool = temp(celcius, 32)

File: functions/notes.c
// Robert Nelson, function notes in C
#include <stdio.h>
/*int num;
char name[50], place[50], verb[50];
int add(int numOne, int numTwo){
    return numOne+numTwo;
}
const char* word(type){
    char choice[50];
    printf("Please give me a %s:\n", type);
    scanf("%s", choice);
    return choice;
}
*/
void due(char assignment[50], char day[20]){
    printf("The %s assigment is due %s\n", assignment, day);
}
/**/

int main(void){
  /*
    printf("Please tell me a number:\n");
    scanf("%d", num);
  
    add(num,10);
    add(4,15);
    add(4,20);
    */
    due("Function Notes", "Today");
    due("Hello World Update", "Tommorow");
    due("Financial Calculator", "Friday");
    return 0;
}

